online-shop/src/app/state/cart/cart.actions.ts

```diff
@@ -0,0 +1,15 @@
+import { createAction, props } from '@ngrx/store';
+import { Action } from '@ngrx/store';
+import { HttpErrorResponse } from '@angular/common/http';
+
+
+export const checkout = createAction(
+  'Checkout'
```

Comment:
Suggestion: Maybe try and keep consistency in the naming of the arguments: ('[Checkout] Checkout')